### DISCUSSION TITLE
Promote Project on different Channels

### DISCUSSION THREAD

**Discussion by ZigRazor** [ZigRazor/CXXGraph#432] [General]
Hi @sbaldu @nolankramer @nrkramer, I think that in this moment we need to promote the project on different channels.
We should write some blog post on the more used platform like [medium](https://medium.com/) or [DEV.to](https://dev.to/) or [CodeProject](https://www.codeproject.com/).
We should also try to publish somethings to [HackerNews](https://news.ycombinator.com/), or some forum like Reddit.
In short, we need to spread the word!
This to increase the community over this now mature project.

What do you think?
You can help me?

**Comment by ZigRazor** [comment]
I wold like to see all the contribution to spread the word in this discussion.

**Comment by sbaldu** [comment]
Personally I think that this is a great idea. You can count on me for help with writing the blogs.

**Reply by ZigRazor** [reply]
I think you can publish independently the article where you prefer

**Reply by ZigRazor** [reply]
@sbaldu do you have write somethings?
Thank you in advance

**Reply by sbaldu** [reply]
Hi. I'm currently busy with finishing my master's thesis, but I'll get on it as soon as I can.

**Reply by ZigRazor** [reply]
Thank You so much!

**Comment by nolankramer** [comment]
Sadly I don't have the time to write blogs/articles, but I'm happy to share the project on LinkedIn and promote articles written by @sbaldu. I'm also happy to find some graph library aggregation posts and promote CXXGraph there.

**Reply by ZigRazor** [reply]
Thank you in advance!

**Comment by badumbatish** [comment]
i can try writing a blog on how to use cxxgraph

**Reply by ZigRazor** [reply]
You are welcome @badumbatish !!!
Thank you in advance

**Reply by ZigRazor** [reply]
@badumbatish do you have write somenthings?

**Reply by badumbatish** [reply]
@ZigRazor apologies, i've been a bit busy and now i'm working on another graph library in rust, i hope you check them out :) 
https://github.com/badumbatish/lean_graph

**Comment by ZigRazor** [comment]
Also Spread the word on social network like Linkedin or Twitter is good.
